### PR TITLE
fix the get tweet json api

### PR DESCRIPTION
**Pull request by JavierJia**
@JeremyLi28 fix the server to get sample tweets from twitter api.

### CODE REVIEW

File: neo/app/controllers/Application.scala [hunk 1]
```diff
@@ -59,9 +59,11 @@ class Application @Inject()(val wsClient: WSClient,
     ActorFlow.actorRef(out => UserActor.props(out, cachesActor, USGeoGnosis))
   }
 
-  def tweet(id:String) = Action {
+  def tweet(id:String) = Action.async {
```
Comment (chenlica) [inline neo/app/controllers/Application.scala]: I am new to Scala, and can give superficial comments: can we add comments to the code? :-)

Comment (JavierJia) [inline neo/app/controllers/Application.scala]: I'm very happy to explain Scala code in review comment, but I really don't like comments in the code which usually shows that the code is not clear enough 😢 

This function actually is a Play framework function which sends a request to that composed `url`. `wsClient.url().get` get the response back by return a `Future[WSResponse]`. Then we just wrap up the body of the response with a "OK 200" return code and send back to the client.

Comment (chenlica) [inline neo/app/controllers/Application.scala]: Never mind then :-)

Comment (JavierJia) [inline neo/app/controllers/Application.scala]: It's actually a pretty good practice to learn the code through code review. @JeremyLi28 @xixZ  you are very encouraged to ask questions about the unknown codes 😄

**Comment by JeremyLi28** [comment]
So I can directly call the '/tweet/' route now?

**Comment by JavierJia** [comment]
Yes. I tested it by simply open `localhosts:9000/tweet/715635459397967872`.

**Comment by JeremyLi28** [comment]
LGTM

**Pull request merged**